app/src/main/res/values/strings.xml

```diff
@@ -188,6 +188,7 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="key_page_down">Page Down (Page Down)</string>
     <string name="move_text_one_line_up">Move text one line up</string>
     <string name="move_text_one_line_down">Move text one line down</string>
+    <string name="insert_next_line">Insert line below</string>
```

Comment:
to my understanding, it goes to end of the next line, and inserts nothing

Reply:
It goes to the end of the current line and inserts a new line. This is useful when editing lists or todo.txt. I can insert a line below with one click instead of having to find the line end and clicking enter. In markdown, the list will be automatically updated too.

If you have used vim, this is similar to the `o` command.

Reply:
vim-o Okay - I would name it "Start new line below"